general/1_terminal/Oleg-Levitskiy/task_1.txt

```diff
@@ -0,0 +1 @@
+exa -l *zoola*
```

Comment:
Solid choice of file system explorers.

I also prefer `exa` to `ls`, however, it's not installed on every machine by default. Let's use `ls`, so that it works everywhere.